src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -29,4 +32,33 @@ class OppgaveController(val oppgaveService: OppgaveService) {
         return oppgaveService.getOppgaver()
     }
 
+    @GetMapping("/oppgaver/ikketildelt")
+    fun findIkkeTildelteOppgaver(): List<OppgaveView> {
+        logger.debug("findIkkeTildelteOppgaver is requested")
+        return oppgaveService.searchOppgaver(OppgaveSearchCriteria(erTildeltSaksbehandler = false))
+    }
+
+    @GetMapping("/oppgaver/tildelt/{ident}")
+    fun findTildelteOppgaver(@PathVariable(name = "ident", required = false) ident: String?): List<OppgaveView> {
+        logger.debug("findTildelteOppgaver is requested")
+        return oppgaveService.searchOppgaver(
+            OppgaveSearchCriteria(
+                erTildeltSaksbehandler = true,
+                saksbehandler = ident
+            )
+        )
+    }
+
+    @GetMapping("/oppgaver/tildelt/me")
```

Comment:
hva med "meg" i stedet for me?

Reply:
Det ville vært naturlig, jeg er helt enig, men såvidt jeg har skjønt er det en standard i NAV på å bruke "me" for å signalisere at det hentes ut data for innlogget bruker.

Reply:
Det er jo en GET URL, og skal som sådan kunne caches, men i dette tilfellet kan den jo ikke det. (Det slår meg at jeg kanskje burde lagt på noe cache-invalidering på endepunktet sånn sett..)

Reply:
Pushet opp en endring nå for å disable browsercache

Reply:
Jeg fjerner hele det endepunktet, ifølge Andreas har vi identen i gui.